identity-wallet/src/state/profile_settings/mod.rs

```diff
@@ -9,12 +9,18 @@ use strum::EnumString;
 use ts_rs::TS;
 
 /// ProfileSettings contains all matters concerning the user profile and its settings.
-#[derive(Default, Serialize, Deserialize, Derivative, TS, Clone, PartialEq, Debug)]
+#[derive(Serialize, Deserialize, Derivative, TS, Clone, PartialEq, Debug)]
+#[derivative(Default)]
 #[ts(export)]
 #[serde(default)]
 pub struct ProfileSettings {
     pub locale: Locale,
     pub profile: Option<Profile>,
+
+    // TODO: Current simplified solution for handling a default DID method. Once we have the did-manager implemented, we
+    // should probably come up with a different solution.
+    #[derivative(Default(value = "String::from(\"did:key\")"))]
```

Comment:
Maybe its better to impl default ourselves. This is error prone to typo's

Reply:
Ok, maybe one more clean thing is to use r#"..."# syntax, then you can remove the backslashes.